schema-2.sql

```diff
@@ -0,0 +1,64 @@
+DROP TABLE IF EXISTS owners;
+CREATE TABLE owners(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(20),
+  age INT,
+  PRIMARY KEY(id)
+);
+
+DROP TABLE IF EXISTS species;
+CREATE TABLE species(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  PRIMARY KEY(id)
+);
+
+DROP TABLE IF EXISTS animals;
+CREATE TABLE animals(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  date_of_birth DATE,
+  escape_attempts INT,
+  neutered BOOLEAN,
+  weight_kg DECIMAL,
+  species_id INT REFERENCES species(id),
+  owner_id INT REFERENCES owners(id),
+  PRIMARY KEY(id)
+);
+
+DROP TABLE IF EXISTS vets;
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
+);
+
+DROP TABLE IF EXISTS specialization;
+CREATE TABLE specializations(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  species_id INT REFERENCES species(id),
+  vet_id INT REFERENCES vets(id),
+  PRIMARY KEY(id)
+);
+
+DROP TABLE IF EXISTS visits;
+CREATE TABLE visits(
```

Comment:
### Optional
- [ ] It would be better if we add indexes on our foreign keys or columns, since this is one basic solution for the performance problems. Kindly try them out. For example; `CREATE INDEX ON visits (animal_id);`

Reply:
@addod19 Thank you for the feedback. we will implement the require changes